src/sentry/rules/processor.py

```diff
@@ -13,11 +13,31 @@
 RuleFuture = namedtuple('RuleFuture', ['rule', 'kwargs'])
 
 
+# TODO(dcramer): come up with a clean way to kill this either by renaming
+# the Event.message attribute or updating all plugins (former is better)
+class EventCompatibilityProxy(object):
+    """
+    A proxy which manages the 'message' attribute on an event to safely
+    upgrade legacy notifications.
+    """
+    __class__ = property(lambda x: x._event.__class__)
+
+    def __init__(self, event):
+        self._event = event
+
+    def __getattr__(self, attr):
+        return getattr(self._event, attr)
+
+    @property
+    def message(self):
+        return self._event.get_legacy_message()
```

Comment:
Might be worth warning here, not sure if anybody actually pays attention to those though.